app/javascript/controllers/dropdown_select_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    this.element.dataset.action = [this.element.dataset.action, "dropdown-select#select"].filter((v) => v).join(" ")
```

Comment:
Little bit of JS to immediately select things when you change sorting/filtering dropdowns on podcasts#index and episodes#index.